### DISCUSSION TITLE
Plotting orientation

### DISCUSSION THREAD

**Discussion by Hohniker** [flexcompute/tidy3d#1065] [Q&A]
Is there an easy way in Tidy3D plotting to change which direction is considered vertical and which is horizontal? For example sim.plot(x=0) gives a plot of the cross section of a waveguide with the z-axis vertical and y-axis horizontal, resulting in an annoying "sideways" waveguide.  I'm working with anisotropic materials, and I'd prefer not to bandaid this problem by renaming axes.

I'm hoping for a syntax akin to sim.plot(x=0, z='x', y = 'y') - but this doesn't work.

**Comment by tylerflex** [comment]
Hi @Hohniker thanks for bringing this up. We'll add an option to transpose plots in a future release. 

https://github.com/flexcompute/tidy3d/issues/1072

**Reply by rajat-bob** [reply]
I'm encountering the same issue. Has it been resolved in the latest versions?

**Reply by tylerflex** [reply]
it hasn't been addressed yet. We are tracking the issue here:
https://github.com/flexcompute/tidy3d/issues/1072
